File: tests/fixtures/sample.swift
import Foundation

struct Greeting {
    let name: String
    let age: Int
    
    func message() -> String {
        return "Hello, \(name)! You are \(age) years old."
    }
}

let g = Greeting(name: "World", age: 25)
print(g.message())

File: tests/fixtures/sample.rb
# Sample Ruby file
class Greeting
  attr_reader :name, :age

  def initialize(name, age)
    @name = name
    @age = age
  end

  def message
    "Hello, #{@name}! You are #{@age} years old."
  end
end

g = Greeting.new("World", 25)
puts g.message
